core/probePlugin/src/main/scala/org/virtuslab/ideprobe/Screenshot.scala

```diff
@@ -9,24 +9,22 @@ object Screenshot {
   private val id = new AtomicInteger(0)
 
```

Comment:
Also! while you're testing this change against git-machete-intellij-plugin... pls check whether `-screen` isn't the better option instead of `-root` for `xwd` invocation... I have an impression that screenshots do **not** include dialog windows, maybe this is going to include them

Reply:
https://app.circleci.com/pipelines/github/VirtusLab/git-machete-intellij-plugin/5245/workflows/dbb72a59-605f-4ac6-b9be-349abdb8cbf6/jobs/5519/artifacts
This artefact is created with `-root` option and some dialog windows are included (I can't see here e.g. rebase dialogs and I do not know why). I tried with -screen option and but that doesn't work for the moment